MiniApp/Classes/MiniApp.swift

```diff
@@ -96,3 +94,18 @@ public class MiniApp: NSObject {
         return realMiniApp.createMiniApp(appInfo: appInfo, completionHandler: completionHandler, messageInterface: messageInterface)
     }
 }
+
+// MARK: - Testing
+public extension MiniApp {
+    /// Create a Mini App for the given mini url.
+    /// Mini app will NOT be downloaded and cached in local, its content will be read directly from provided url.
+    /// This should only be used for previewing a mini app from a local server.
+    ///
+    /// - Parameters:
+    ///   - url: a HTTP url containing Mini App content
+    /// - Returns: MiniAppDisplayProtocol: Protocol that helps the hosting application to communicate with the displayer module of the mini app. More like an interface for host app
+    ///                         to interact with View component of mini app.
+    func create(url: URL, messageInterface: MiniAppMessageDelegate) -> MiniAppDisplayProtocol {
+        return realMiniApp.createMiniApp(url: url, messageInterface: messageInterface)
```

Comment:
Can we have something like a completionHandler or anything that would let the host app know if it fails to load the URL.

Reply:
I thought about that. I'd have to pass the handler all the way to `MiniAppWebView`. That'll complicate the code a little bit but I can do that

Reply:
Yeah, I know. This looks cleaner, but we have to let the host app know if something goes wrong